### PR TITLE
migrations complated

### PR DESCRIPTION
**Pull request by Dilshod0609**

### CODE REVIEW

**Comment by makhsam** [comment]
@Mirvali97 Mirvali tekshirib berin

**Comment by makhsam** [comment]
@Mirvali97 Mirvali tekshirib berin

**Comment by Mirvali97** [comment]
Hop

File: database/migrations/2021_02_03_230003_create_user_sessions_table.php [hunk 1]
```diff
@@ -15,6 +15,12 @@ public function up()
     {
         Schema::create('user_sessions', function (Blueprint $table) {
             $table->id();
+            $table->foreignId('user_id');
+            $table->ipAddress('ip_address');
```
Comment (Mirvali97) [inline database/migrations/2021_02_03_230003_create_user_sessions_table.php:19]: int('ipAddress ') bulsa kerak menimcha @makhsam

File: database/migrations/2021_02_03_230055_create_user_tests_table.php [hunk 2]
```diff
@@ -15,6 +15,13 @@ public function up()
     {
         Schema::create('user_tests', function (Blueprint $table) {
             $table->id();
+            $table->foreignId('user_id');
+            $table->foreignId('exam_id')->constrained('user_exams');
```
Comment (Mirvali97) [inline database/migrations/2021_02_03_230055_create_user_tests_table.php:19]: >foreignId('exam_id')->constrained('user_exams');
agar // exam id or null bulsa nullable() olish kerak menimcha @makhsam

**Review by Mirvali97 — COMMENTED** [review]
@makhsam  Aka hammasi zo'r chiqqan  2 ta joyda tahminiy hatolikni topdim lekin bar bir sizham tekshirib bersangiz yaxshi bo'lar edi

File: database/migrations/2021_02_03_230003_create_user_sessions_table.php (hunk 1 above)
Comment (makhsam) [reply database/migrations/2021_02_03_230003_create_user_sessions_table.php:19]: [Laravel documentation](https://laravel.com/docs/8.x/migrations#column-method-ipAddress) da shunaqa diyilgan ekan:
> The `ipAddress` method creates an `INTEGER` equivalent column

Mana bunday ko'rinishda saqlash kerak:
```php
$table->string('ip_address', 45);
```

Comment (makhsam) [reply database/migrations/2021_02_03_230003_create_user_sessions_table.php:19]: @Mirvali97 gapiz to'g'ri

Comment (Mirvali97) [reply database/migrations/2021_02_03_230003_create_user_sessions_table.php:19]: @makhsam  ha to'gri uzimham adashibman 👍

File: database/migrations/2021_02_03_230055_create_user_tests_table.php (hunk 2 above)
Comment (makhsam) [reply database/migrations/2021_02_03_230055_create_user_tests_table.php:19]: To'g'ri `exam_id` nullable bo'lishi kerak. @Dilshod0609 19-line ni quyidagiga o'zgartiring:

```php
$table->foreignId('exam_id')->nullable();
$table->foreign('exam_id')->references('id')->on('user_exams');
```

**Review by makhsam — COMMENTED** [review]
@Dilshod0609 migration yozganizda `nullable` fieldlaga e'tibor berin. Keyin ko'p joyda foreign constraintlar ishlatilinmagan ekan:
https://laravel.com/docs/8.x/migrations#foreign-key-constraints

**Review by makhsam — COMMENTED** [review]
@Dilshod0609 shulani ko'rib chiqin

**Review by makhsam — APPROVED** [review]
Barchaga rahmat!

**Pull request merged**